content/hi/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,129 @@
+---
+# reviewers:
+# - lavalamp
+title: कुबेरनेट्स घटक
+content_type: concept
+description: >
+  कुबेरनेट्स क्लस्टर में कंट्रोल प्लेन के घटक और मशीनों का एक समूह होता है।
+  विमान और मशीनों का एक सेट होता है जिसे नोड्स कहा जाता है।
+weight: 30
+card:
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+जब आप कुबेरनेट्स को डिप्लॉय करते हैं, तो आपको एक क्लस्टर मिलता है।
+{{< glossary_definition term_id="cluster" length="all" prepend="कुबेरनेट्स क्लस्टर में">}}
+
+यह दस्तावेज़ आपके लिए आवश्यक विभिन्न घटकों की रूपरेखा देता है
+एक पूर्ण और कार्यशील कुबेरनेट क्लस्टर।
+
+{{< figure src="/images/docs/components-of-kubernetes.svg" alt="Components of Kubernetes" caption="The components of a Kubernetes cluster" class="diagram-large" >}}
```

Comment:
The alt attribute could be in Hindi (and could explain, in Hindi, that the text is in English)

Optionally: localize the diagram as well.